WordPress/Classes/Utility/Media/MediaFileManager.swift

```diff
@@ -221,40 +221,36 @@ class MediaFileManager: NSObject {
     /// Removes any local Media files, except any Media matching the predicate.
     ///
     fileprivate func purgeMediaFiles(exceptMedia predicate: NSPredicate, onCompletion: (() -> Void)?, onError: ((Error) -> Void)?) {
```

Comment:
This could _could_ benefit from using a `Result<Void, Error> -> Void` callback. However, it's used only once; updating it at this time would have diminishing returns.